task_processing/plugins/mesos/translator.py

```diff
@@ -2,17 +2,17 @@
 
 # https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto
 MESOS_TASK_STATUS_TO_EVENT = {
-    0: Event(platform_type='starting'),
-    1: Event(platform_type='running'),
-    2: Event(platform_type='finished'),
-    3: Event(platform_type='failed'),
+    0: Event(platform_type='starting', terminal=False),
```

Comment:
why? I don't think there is a reasonable default - I'd leave it to the implementer to decide either way per Event?